src/main/java/com/sandvoxel/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -88,10 +87,7 @@
     private int fuelTotal = 0;
     private Item lastFuelType;
     private int lastFuelValue;
-    private float speedMultiplier = 0.0f;
     private float fortuneMultiplier = 0.5f;
-    private int currentTotalProcessTime = 0;
-    private boolean test = true;
 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "speedMultiplier" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS1068)